octoprint_SpoolManager/templates/SpoolManager_tab_dialogs.jinja2

```diff
@@ -281,6 +306,7 @@
                                     <div><sup>2</sup><small>Needed for filament usage prediction</small></div>
                                     <div><sup>3</sup><small>See <a href="https://bitfab.io/blog/3d-printing-materials-densities/" target="newTab">https://bitfab.io/blog/3d-printing-materials-densities/ (without warranty)</a></small></div>
                                     <div><sup>4</sup><small>Not send to the printer</small></div>
+                                    <div><sup>5</sup><small>Temperature offset enabled via Plugin Settings -> temperature tab</small></div>
```

Comment:
Btw. Why is this enabled globally and not in each single spool?

Reply:
I thought it makes more sense as a global setting because that's how I use it. Setting this globally makes the plugin look for the setting and turns off the offset if it isn't configured on the spool.